### PR TITLE
style improvements

### PR DESCRIPTION
**Pull request by Guntrisoft**
- redone BETA logo to include text as a shape in the SVG
    - ensures that logo scales correctly with the BETA
      text
- altered column layout slightly on article pages
  so that it fits better at iPad widths
- altered grey background on homepage to start
  halfway down the campaign pane box

### CODE REVIEW

**Comment by Guntrisoft** [comment]
@benbarnett

File: app/assets/stylesheets/components/article_body/_inline_callout.scss [hunk 1]
```diff
@@ -6,6 +6,7 @@
   background-size: 25px;
   padding: baseline-unit(4);
   margin: baseline-unit(6) 0;
+  max-width: 668px;
```
Comment (benbarnett) [inline app/assets/stylesheets/components/article_body/_inline_callout.scss]: Worth making `668px` a variable as its used twice?

File: app/assets/stylesheets/components/article_body/_next_steps.scss [hunk 2]
```diff
@@ -5,6 +5,7 @@
   padding-top: baseline-unit(9);
   margin-top: baseline-unit(12);
   border-radius: 20px;
+  max-width: 668px;
```
Comment (benbarnett) [inline app/assets/stylesheets/components/article_body/_next_steps.scss]: As above

File: app/assets/images/beta.svg [hunk 3]
```diff
@@ -0,0 +1,41 @@
+<?xml version="1.0" encoding="utf-8"?>
```
Comment (benbarnett) [inline app/assets/images/beta.svg]: Is this file needed?

**Comment by benbarnett** [comment]
:+1: after those comments
